mobify/platform-scaffold
# Include AMP links for appropriate pages

...so GoogleBot will add AMP search results

~~**JIRA**: (link to JIRA ticket)~~
 **Linked PRs**: #925

## Changes
- Add AMP url checking + link tag inclusion

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Observe a link tag with `amphtml` exists for PLPs and PDPs, on both mobile and desktop.
